### DISCUSSION TITLE
Is the localization feature permanently removed?

### DISCUSSION THREAD

**Discussion by NANDOHAD** [vladmandic/sdnext#548] [General]
I am very sorry to write English with automatic translation.
We cannot migrate from AUTOMATIC1111 as we do not have localization capabilities.
And humbly thank you.

**Comment by vladmandic** [comment]
It is unless someone wants to contribute and take ownership of that part of the code - I cannot maintain it myself.

**Reply by NANDOHAD** [reply]
understood. thank you.
